sympy/physics/mechanics/tests/test_actuator.py

```diff
@@ -849,3 +853,144 @@ def test_to_loads(self):
                 diff = (calculated_component - expected_component).subs(substitutions).evalf()
                 # Check if the absolute value of the difference is below a threshold
                 assert Abs(diff) < 1e-9, f"The forces do not match. Difference: {diff}"
+
+class TestCoulombKineticFriction:
+    @pytest.fixture(autouse=True)
+    def _block_on_surface(self):
+        r"""A block sliding on a surface.
+
+        Notes
+        =====
+        This test validates the correctness of the CoulombKineticFriction by simulating
+        a block sliding on a surface with the Coulomb kinetic friction force.
+        The test covers scenarios with both positive and negative velocities.
+
+        Equations of Motion
+        ===================
+        The equations of motion for the block are derived considering
+        the applied force F, the mass m, the gravitational constant g, and the friction forces
+        as described above.
+
+        The general form of the equation of motion is:
+
+            F - F_friction = m * xdd
```

Comment:
What is `F`?

Reply:
Here you assume that the first point in the pathway is not moving in the inertial reference frame and that v and F_friction are defined with sign conventions such that they are in the same direction. That doesn't seem like the same convention you used in the docstring of the new actuator.